Bottom: f5a6e0cc0bdb8da2ddcecec1e29d614f68473e7f
Top:    518dbf0e02a2bd0f3116af3009c0693be65fb547
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-14 18:23:06 +0000

Refresh of avoid-clearing-pins-vias-from-

---

diff --git a/src/draw.c b/src/draw.c
index e15cfc4..9035f31 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1944,8 +1944,6 @@ DrawRegularText (LayerTypePtr Layer, TextTypePtr Text, int unused)
 void
 DrawPolygon (LayerTypePtr Layer, PolygonTypePtr Polygon, int unused)
 {
-  int layernum;
-
   if (TEST_FLAG (SELECTEDFLAG | FOUNDFLAG, Polygon))
     {
       if (TEST_FLAG (SELECTEDFLAG, Polygon))
@@ -1955,7 +1953,6 @@ DrawPolygon (LayerTypePtr Layer, PolygonTypePtr Polygon, int unused)
     }
   else
     gui->set_color (Output.fgGC, Layer->Color);
-  layernum = GetLayerNumber (PCB->Data, Layer);
   DrawPolygonLowLevel (Polygon);
 }
